android/app/src/main/jni/Application.mk

```diff
@@ -0,0 +1,2 @@
+APP_ABI := armeabi
```

Comment:
Should be `GSTREAMER_ROOT_ARM` -> `GSTREAMER_ROOT_ANDROID_ARM`. So we always build all architectures, good approach I think. :)